lte/gateway/c/oai/lib/itti/intertask_interface.c

```diff
@@ -444,8 +444,11 @@ int itti_send_msg_to_task(
       /*
        * Enqueue message in destination task queue
        */
-      lfds710_queue_bmm_enqueue(
+      rv = lfds710_queue_bmm_enqueue(
         &itti_desc.tasks[destination_task_id].message_queue, NULL, new);
+      AssertFatal(
+	rv,
+	"no queue place for task id %d\n", destination_task_id);
```

Comment:
There is an ongoing effort to eliminate AssertFatal statements to prevent process exits. @ssanadhya can you check this?

Reply:
Yes, our goal is to minimize the number of AssertFatals, and use error logging as much as possible. In this particular case, it will be better to log an error instead of crashing the service.

Reply:
I believe it is better to have AssertFatal's in debug compilation and replace them with logging only in production. Fail more, fail sooner.